native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe.Tests/User.cs

```diff
@@ -0,0 +1,41 @@
+using Avro;
+using Avro.Specific;
+
+namespace AWSGsrSerDe.Tests
+{
+    public class User : ISpecificRecord
+    {
+        
+        private static readonly Schema UserSchema = Schema.Parse("{\"type\":\"record\",\"name\":\"User\",\"namespace\":\"AWSGsrSerDe.Tests\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"favorite_number\",\"type\":[\"int\",\"null\"]},{\"name\":\"favorite_color\",\"type\":[\"string\",\"null\"]}]}");
+
+        public string Name;
+        public int FavoriteNumber;
+        public string FavoriteColor;
+
```

Comment:
Are getters / setters created for these?

Reply:
I have made them public since this class is only for testing purpose. Do you think we need a getter/setter in this case?